app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -369,7 +369,7 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
   )
 
   useEffect(() => {
-    if (route.params?.autoValidate && route.params.payment) {
+    if (route.params.payment) {
```

Comment:
@nicolasburtey right here autoValidate is defaulting to false for the linking and so it is not receiving the payment param.